.sync.yml

```diff
@@ -13,7 +13,11 @@
     - /provision.yaml
 Rakefile:
   default_disabled_lint_checks:
-    - '140chars'
+    - relative
+    - parameter_types
+    - parameter_documentation
+    - documentation
```

Comment:
does the documentation check actually exist?

Reply:
confirmed
```
pdk bundle exec puppet-lint --help | grep documentation
pdk (INFO): Using Ruby 3.2.2
pdk (INFO): Using Puppet 8.4.0
        --no-documentation-check     Skip the documentation check.
```